speech-to-text/v1.ts

```diff
@@ -225,4 +231,66 @@ class SpeechToTextV1 extends GeneratedSpeechToTextV1 {
   }
 }
 
+namespace SpeechToTextV1 {
+  export type Callback<T> = (err: Error | SpeechToTextError | null, res?: T) => void;
+
+  export interface SpeechToTextError extends Error {
+    message: string;
+    code?: string;
```

Comment:
> had to make due with a weaker interface design with optional code even though it should never actually be undefined in usage.

I think that's fine. It can always be revisited in the future but I think that's a very small issue